### ISSUE TITLE
HaProxy error 'No enabled listener found' when running docker-compose under Docker 1.12

### ISSUE THREAD

**Issue by fbuchinger** [TeamHG-Memex/aquarium#5]
When running the docker-compose.yml generated by aquarium, I get an haproxy error under Docker 1.12:
```
$ docker-compose up
Starting aquarium_splash1_1
Starting aquarium_splash2_1
Starting aquarium_splash0_1
Starting aquarium_haproxy_1
Attaching to aquarium_splash1_1, aquarium_splash2_1, aquarium_splash0_1, aquarium_haproxy_1
[35mhaproxy_1  |[0m <7>haproxy-systemd-wrapper: executing /usr/local/sbin/haproxy -p /run/haproxy.pid -f /usr/local/etc/haproxy/haproxy.cfg -Ds
[35mhaproxy_1  |[0m [ALERT] 009/202127 (6) : [/usr/local/sbin/haproxy.main()] No enabled listener found (check for 'bind' directives) ! Exiting.
```
SO has a [related question](http://stackoverflow.com/a/33343303/113936) which indicates that there might be a path error. Any ideas?

Here is my docker info:
```
$ docker version
Client:
 Version:      1.12.2
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   bb80604
 Built:        Tue Oct 11 17:00:50 2016
 OS/Arch:      windows/amd64

Server:
 Version:      1.12.0
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   8eab29e
 Built:        Thu Jul 28 23:54:00 2016
 OS/Arch:      linux/amd64
```

**Comment by fbuchinger** [comment]
docker-compose seems to have problems with Windows drives other than C:. Initially, I created the aquarium project on my E: drive and got the error above. When I copied the project to C: and ran `docker-compose up` again, everything worked fine. Windows users should also pay attention to a breaking change in [docker-compose 1.9](https://github.com/docker/compose/releases/tag/1.9.0): windows paths for volumes are no longer out-converted.
